### PR TITLE
Issue 3

### PR DESCRIPTION
**Pull request by Sabnock01**
PR for ERC20 and Ether wrappers and their respective tests.

### CODE REVIEW

File: src/lesson_3/EtherWrapper.sol [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    ///@notice Event emitted when tokens are wrapped
+    event Wrapped(address indexed from, uint256 amount);
+    ///@notice Event emitted when tokens are unwrapped
+    event Unwrapped(address indexed to, uint256 amount);
+
+    ///@notice Creates a wrapper for a given ERC20 token
+    ///@param name the token's name
+    ///@param symbol the token's symbol
+    ///@dev the token will have 18 decimals as specified by the constructor in ERC20Mock
+    constructor(string memory name, string memory symbol) ERC20Mock(name, symbol) {}
```
Comment (alcueca) [inline src/lesson_3/EtherWrapper.sol:21]:
```suggestion
    constructor(string memory name, string memory symbol) ERC20(name, symbol) {}
```
As above.

**Review by alcueca — CHANGES_REQUESTED** [review]
Only the small oversight about ERC20Mock. Otherwise great clean code.

File: src/lesson_3/EtherWrapper.sol [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    ///@notice Wraps Ether
+    ///@dev emits Wrapped event when transfer successful and error message upon failure
+    receive() external payable {
```
[the remark this answers is not in the document]
Comment (Sabnock01) [reply src/lesson_3/EtherWrapper.sol]: Believe I addressed this.

**Review by devtooligan — APPROVED** [review]
lgtm

**Pull request merged**